contrib/templates/data-engineering/README.md

```diff
@@ -6,5 +6,5 @@ assets in DABs.
 Install it using
 
 ```
-databricks bundle init https://github.com/databricks/bundle-examples --template-dir contrib/templates/data-engineering/base
+databricks bundle init https://github.com/databricks/bundle-examples --template-dir contrib/templates/data-engineering/base --branch data-engineering
```

Comment:
For development purposes, 596ece0174b405e4276d11e97ca9580a47b6b11c adds `--branch`; this should be reverted before the PR is merged.